public/js/index.js

```diff
@@ -1474,16 +1475,45 @@ function selectRange($list, $select, $from) {
     changeHash('#' + $select.attr('rel'));
 }
 
-$(function () {
-    if ($('.user.signin').length > 0) return;
-    $('form').areYouSure();
+function initDashboardSearch() {
+    var el = document.getElementById('dashboard-repo-search');
+    if (!el) {
+        return;
+    }
+
+    new Vue({
+        delimiters: ['<%', '%>'],
+        el: el,
+        data: {
+            repos: [],
+            searchQuery: ''
+        },
+        mounted: function() {
+            this.searchRepos();
+        },
+        methods: {
+            searchKeyUp: function() {
+                this.searchRepos();
+            },
+
+            searchRepos: function() {
+                var self = this;
+                $.getJSON('/api/v1/repos/search?q=' + self.searchQuery, function(result) {
```

Comment:
And all of sudden I feel the urge to write an API-client for javascript (I hate hard-coded URLs 😆 )